GitCommands/Settings/AppSettings.cs

```diff
@@ -1688,6 +1688,18 @@ public static bool BlameShowOriginalFilePath
             set => SetBool("Blame.ShowOriginalFilePath", value);
         }
 
+        public static bool ContinuousScrollToNextFileOnlyWithAlt
+        {
+            get => GetBool("continuousscrolltonextfileonlywithalt", false);
+            set => SetBool("continuousscrolltonextfileonlywithalt", value);
+        }
+
+        public static int AutoViewScrollDelay
+        {
+            get => GetInt("autoviewscrolldelay", 600);
+            set => SetInt("autoviewscrolldelay", value);
+        }
```

Comment:
```suggestion
        public static int AutomaticContinuousScrollDelay
        {
            get => GetInt("DiffViewer.AutomaticContinuousScrollDelay", 600);
            set => SetInt("DiffViewer.AutomaticContinuousScrollDelay", value);
        }
```

This setting is not exposed anywhere, why are we making it a setting?

Reply:
I suppose I can see the value in this.